lib/vm.js

```diff
@@ -32,7 +32,7 @@ const {
   ERR_INVALID_ARG_TYPE,
   ERR_VM_MODULE_NOT_MODULE,
 } = require('internal/errors').codes;
-const { isModuleNamespaceObject, isArrayBufferView } = require('util').types;
+const { isModuleNamespaceObject, isArrayBufferView } = require('internal/util/types');
```

Comment:
Hint: this line shouldn't exceed 80, so we need to have newline here.

```suggestion
const {
  isArrayBufferView,
  isModuleNamespaceObject
} = require('internal/util/types');
```

Reply:
👍